Cargo.toml

```diff
@@ -184,6 +184,7 @@ raft = { git = "https://github.com/tikv/raft-rs", branch = "master", default-fea
 raft-proto = { git = "https://github.com/tikv/raft-rs", branch = "master", default-features = false }
 protobuf = { git = "https://github.com/pingcap/rust-protobuf", rev = "65e9df20fbcbcf2409d5ee86a2332ecd04c534f8" }
 protobuf-codegen = { git = "https://github.com/pingcap/rust-protobuf", rev = "65e9df20fbcbcf2409d5ee86a2332ecd04c534f8" }
+grpcio = { path = "/home/lxt/repo/grpc-rs", default-features = false, features = ["openssl-vendored"] }
```

Comment:
The main GRPC changes have been merged upstream now.

Reply:
I saw that and waiting https://github.com/tikv/grpc-rs/pull/503

Reply:
Ping